docs/Manual/09 Parameters.md

```diff
@@ -3,10 +3,10 @@
 
 # Section 9: Parameters
 
-Accera parameters are placeholders that get replaced with concrete values when we add a function to a package. A parameter can be used in a `Nest`, a `Schedule`, or an `Plan`.
+Accera's parameters are placeholders that get replaced with concrete values when adding a function to a package. A parameter can be used in a `Nest`, a `Schedule`, or a `plan`.
 
 ## Parameterized nests
-Recall that a `Nest` represents the loop-nest logic. We can parameterize the nest shape and the iteration logic. For example, consider the following parameterized version of matrix multiplication:
+Recall that a `Nest` represents the loop-nest logic. We can parametrize the nest shape and iteration logic. For example, consider the following parameterized version of matrix multiplication:
```

Comment:
Maybe clearer to state: "We can parameterize the nest's shape and iteration logic."

(Also, "parameterize" rather than "parametrize" for consistency)